### DISCUSSION TITLE
Live Preview

### DISCUSSION THREAD

**Discussion by Daynlight** [chriskthomas/linkfree-generator#39] [Ideas]
Add Tab for see preview file that will be created.
![Zrzut ekranu 2024-06-26 185045](https://github.com/chriskthomas/linkfree-generator/assets/62157770/c49bb549-3262-4a35-bf38-8a631864cd8a)

**Comment by chriskthomas** [comment]
That would be cool. We could have the preview option submit to an iframe on desktop. The constant re-submissions would be a little rough if the user uploads a photo. I'm not sure how to get around that without re-implementing the template in the frontend.

**Reply by Daynlight** [reply]
I will try
